src/Buckets/BucketCalc.ts

```diff
@@ -15,39 +16,40 @@ export class BucketCalc {
     this.requestFactory = requestFactory;
   }
 
-  public async getBuckets(): Promise<IBuckets> {
+  public async getBuckets(): Promise<Bucket[]> {
     const latest: IBlock = await this.util.getBlock('latest');
-    return {
-      currentBuckets: await this.getCurrentBuckets(latest),
-      nextBuckets: await this.getNextBuckets(latest),
-      afterNextBuckets: await this.getAfterNextBuckets(latest)
-    };
+
+    const currentBuckets = await this.getCurrentBuckets(latest);
+    const nextBuckets = await this.getNextBuckets(latest);
+    const afterNextBuckets = await this.getAfterNextBuckets(latest);
+
+    return currentBuckets.concat(nextBuckets).concat(afterNextBuckets);
```

Comment:
This line is completely okay, but worth to mention it would be also possible to do:

```suggestion
    return return [...currentBuckets, ...nextBuckets, ...afterNextBuckets];
```

Reply:
New-school vs old-school 👍